en/components/date_range_picker.md

```diff
@@ -0,0 +1,431 @@
+
+When in dropdown mode, the `Done` button is not available. In both modes value is set when dates are picked from the calendar.
+
+#### Keyboard Navigation
+> Note: Use the demos for [Drop down mode](#dropdown-mode) and [Separate Editable Inputs](#separate-editable-inputs) to try the keyboard combinations defined in that topic section.
+
+To open/close [`IgxDateRangePickerComponent`]({environment:angularApiUrl}/classes/igxdaterangepickercomponent.html) calendar, using keyboard, the [`mode`]([`IgxDateRangePickerComponent`]({environment:angularApiUrl}/classes/igxdaterangepickercomponent.html#mode)) should be `dropdown`.
+
+- <kbd>Alt</kbd> + <kbd>Down Arrow</kbd> - Opens the drop-down calendar and focuses it
+- <kbd>Alt</kbd> + <kbd>Up Arrow</kbd> - Closes the drop-down and focuses the input field (start input in two inputs mode)
+- <kbd>Esc</kbd> - Closes the drop-down and focuses the input field (start input in two inputs mode)
+
+Keyboard navigation for the Date Range Picker calendar is available for all the modes and configurations. When calendar is opened it is focused and the following keyboard combinations can be used:
+
+- <kbd>Enter</kbd> selects start and end dates
+- <kbd>PageUp</kbd>, <kbd>PageDown</kbd>, <kbd>Shift</kbd> + <kbd>PageUp</kbd>, <kbd>Shift</kbd> + <kbd>PageDown</kbd>, <kbd>Home</kbd>, <kbd>End</kbd>, <kbd>Tab</kbd> - to navigate through calendar
+
+The [calendar keyboard navigation](calendar.md#keyboard-navigation) section contains all keyboard combinations that can be applied to the calendar.
+
+When two separate inputs are defined then [keyboard navigation](date_time_editor.md#keyboard-navigation) of the [`igxDateTimeEditor`]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html) directive are also applicable for the Date Range Picker component:
+
+- <kbd>Ctrl</kbd> / <kbd>Cmd</kbd> + <kbd>Arrow Left</kbd> / <kbd>Right</kbd> - navigates between date sections. On <kbd>Ctrl</kbd> / <kbd>Cmd</kbd> + <kbd>Right</kbd> it goes to the end of the section. If already there it goes to the end of next section if any. It works in a similar fashion in the opposite direction.
+- <kbd>Arrow Up</kbd> / <kbd>Down</kbd> - increment/decrement date portions.
+- <kbd>Ctrl</kbd> / <kbd>Cmd</kbd> + <kbd>;</kbd> - set the current day and time in the editor.
+
+<div class="divider--half"></div>
+
+#### Formatting
+
+Date Range picker supports different display and input formats.
+
+The display format value can be one of the defined by Angular [DatePipe](https://angular.io/api/common/DatePipe), which allows it to support predefined format options, such as `shortDate` and `longDate`.
+
+Input Format doesn't support predefined format options, such as `shortDate` and `longDate`, but accepts a constructed format string using characters supported by the DatePipe, e.g. `MM/dd/yyyy`. If there isn't input format defined then [Angular locale ID](https://angular.io/api/core/LOCALE_ID) token is used for a building an input format.
```

Comment:
Start positive like this:
`Input Format support this and that. Note it does not support those.`
and not
`Input Format does not support those, but supports that.`